internal/services/topic.go

```diff
@@ -144,11 +145,18 @@ func (ts *TopicService) alterTopic(currentPartitions int, brokersNumber int) (ma
 	var err error
 	// less partitions than brokers (scale up)
 	if currentPartitions < brokersNumber {
-		// passing the assigments just for the partitions that needs to be created
-		err = ts.admin.CreatePartitions(ts.canaryConfig.Topic, int32(brokersNumber), ass[currentPartitions:], false)
+		// when replication factor is less than 3 because brokers are not 3 yet (see replicationFactor := min(brokersNumber, 3)),
+		// it's not possible to create the new partitions directly with a replication factor higher than the current ones.
+		// So first alter the assignment of current partitions with new replicas (higher replication factor)
+		// fixes: https://github.com/strimzi/strimzi-canary/issues/16
```

Comment:
I don't think we need comments in the code about (what will become) fixed issues. Just describe this case if necessary.